### PR TITLE
#924 improve layout form fields

### LINKED ISSUE #924: Improve Layout of Form Fields - Work Package Page Edit Mode
### Description

We want to update the layout of all the forms to be cleaner, see specific details below.

Epic: #899 

### Acceptance Criteria

- duration can be smaller if it's easy
- duration can go to the right of start date
- project lead and project manager should moreso match how the project edit mode ones look
- dependencies can be added and aligned horizontally instead of vertically
- dependencies should also probably have an error message when they aren't WBS numbers
- same note about bullets being full width
- breadcrumb project name

### Proposed Solution

N/A

### Mocks

image

### PR DESCRIPTION
**Pull request by Aaryan1203**
Repositioned the fields and breadcrumbed the project name. I originally had the dependencies being added horizontally but it didn't look too good so I kept them vertical and instead just added some margin between each of them.

Screenshot 2023-03-25 at 5 58 45 PM

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #924

### CODE REVIEW

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx [hunk 1]
```diff
@@ -51,8 +51,8 @@ const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) =>
             />
           </FormControl>
         </Grid>
-        <Grid item xs={12} md={'auto'} sx={{ mt: 1 }}>
-          <FormControl sx={{ width: '90%' }}>
+        <Grid item xs={12} md={3} sx={{ mt: 1 }}>
+          <FormControl sx={{ width: '95%' }}>
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx:55]: i would make these all full width and then add spacing between the grid items

File: src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx [hunk 2]
```diff
@@ -72,7 +72,23 @@ const WorkPackageEditDetails: React.FC<Props> = ({ users, control, errors }) =>
...
+        </Grid>
+        <Grid item xs={12} md={1} sx={{ my: 1 }}>
+          <FormControl>
+            <FormLabel>Duration</FormLabel>
+            <ReactHookTextField
+              name="duration"
+              control={control}
+              type="number"
+              placeholder="Enter duration..."
+              sx={{ width: '140%' }}
```
Comment (Peyton-McKee) [inline src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditDetails.tsx:86]: huh? why 140% width

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
good start, make sure to test all the different widths it could be because at certain points the label text wraps and it doesnt look great because theyre not all inline, you should just be able to add the nowrap prop to those things i believe, also when its small duration doesnt take up the entire screen when it should

**Review by Peyton-McKee — CHANGES_REQUESTED** [review]
for the start date wrapping look up css ways to nowrap and truncate form labels in mui

**Review by Peyton-McKee — APPROVED** [review]
LGTM

**Review by anthonybernardi — APPROVED** [review]

**Pull request merged**